src/components/MultiComboBox/useListbox.tsx

```diff
@@ -0,0 +1,316 @@
+                className={activeClass}
+                id={addingButtonId}
+                role="option"
+              >
+                <AddIcon
+                  name="fa-plus-circle"
+                  size={14}
+                  color={theme.palette.TEXT_LINK}
+                  $theme={theme}
+                />
+                <AddText themes={theme}>「{label}」を追加</AddText>
+              </AddButton>
+            )
+          }
+          return (
+            <SelectButton
+              key={`item-${label}`}
+              type="button"
+              themes={theme}
+              disabled={disabled}
+              onClick={() => onSelect(option)}
+              onMouseOver={() => setActiveOptionIndex(i)}
+              className={activeClass}
+              id={getOptionId(option)}
+              role="option"
+            >
+              {label}
+            </SelectButton>
+          )
+        })}
+
+        {isDuplicated && (
+          <NoItems themes={theme} aria-live="polite">
+            重複する選択肢は追加できません
+          </NoItems>
+        )}
+
+        {hasNoMatch && (
+          <NoItems themes={theme} aria-live="polite">
+            一致する選択肢がありません
+          </NoItems>
+        )}
+      </Container>,
+      portalRoot,
+    )
+  }
+  return {
+    renderListbox,
+    setDropdownStyle,
+    resetActiveOptionIndex: () => setActiveOptionIndex(null),
+    handleInputKeyDown,
+    listboxRef,
+    aria: {
+      listboxId,
+      activeDescendant,
+    },
+  }
+}
+
+const Container = styled.div<{ top: number; left: number; width: number; themes: Theme }>(
+  ({ top, left, width, themes }) => {
+    const { size, frame } = themes
+    return css`
+      position: absolute;
+      top: ${top}px;
+      left: ${left}px;
+      overflow-y: auto;
+      max-height: 300px;
+      width: ${width}px;
+      padding: ${size.pxToRem(size.space.XXS)} 0;
+      border-radius: ${frame.border.radius.m};
+      box-shadow: rgba(51, 51, 51, 0.3) 0 4px 10px 0;
+      background-color: #fff;
+      white-space: nowrap;
+      box-sizing: border-box;
+      &.hidden {
+        display: none;
+      }
+      z-index: ${themes.zIndex.OVERLAP};
+    `
+  },
+)
+const NoItems = styled.p<{ themes: Theme }>`
+  ${({ themes }) => {
+    const { size } = themes
+
+    return css`
+      margin: 0;
+      padding: ${size.pxToRem(size.space.XXS)} ${size.pxToRem(size.space.XS)};
+      background-color: #fff;
+      font-size: ${size.font.TALL}px;
+    `
+  }}
+`
+const SelectButton = styled.button<{ themes: Theme }>`
+  ${({ themes }) => {
+    const { size, palette } = themes
+
+    return css`
+      display: block;
+      width: 100%;
+      border: none;
+      padding: ${size.pxToRem(size.space.XXS)} ${size.pxToRem(size.space.XS)};
+      background-color: #fff;
+      font-size: ${size.font.TALL}px;
+      text-align: left;
+      cursor: pointer;
+
+      &.active {
+        background-color: ${palette.COLUMN};
```

Comment:
Safariのユーザースタイルシートで active な時に文字色が白になるっぽいので、 `color: inherit` を追加すると良さそうです。

```suggestion
        background-color: ${palette.COLUMN};
        color: inherit;
```

<img width="720" alt="SafariでListItemをActiveにしたスクリーンショット" src="https://user-images.githubusercontent.com/6724665/101705653-c3571100-3aca-11eb-8575-b21bf397c372.png">

Reply:
@wmoai コレは対応漏れでしょうか？

Reply:
すみません！
30c6933 で対応してます！